### PR TITLE
scripted-diff: use RPCArg::Optional::OMITTED over OMITTED_NAMED_ARG

### PR DESCRIPTION
**Pull request by fanquake**
Remove deprecated `RPCArg::Optional::OMITTED_NAMED_ARG` in favour of `OMITTED`.

### CODE REVIEW

**Comment by MarcoFalke** [comment]
The scripted diff can be:
```sh
sed -i -e "/Deprecated alias for OMITTED, can be removed/d" src/rpc/util.{h,cpp}
sed -i -e "s/OMITTED_NAMED_ARG/OMITTED/g" $(git grep -l "OMITTED_NAMED_ARG" src/)

**Review by kouloumos — COMMENTED** [review]
ACK 92e7a51579712c8108d565a639b206e7555d58e9
I verified that there are no other occurrences of `OMITTED_NAMED_ARG`. I've also verified that there is no diff at the rendered RPC doc after this change by using the RPCdump-patch from https://github.com/bitcoin/bitcoin/pull/14726#pullrequestreview-175423309.
<details><summary>Updated RPCdump-patch</summary>

```diff
diff --git a/src/init.cpp b/src/init.cpp
index 685583bcb..5120d23b5 100644
--- a/src/init.cpp
+++ b/src/init.cpp
@@ -1190,6 +1190,30 @@ bool AppInitMain(NodeContext& node, interfaces::BlockAndHeaderTipInfo* tip_info)
     for (const auto& client : node.chain_clients) {
         client->registerRpcs();
     }
+    JSONRPCRequest jreq;
+    jreq.mode = JSONRPCRequest::GET_HELP;
+    UniValue unused_result;
+    for (const auto& command : tableRPC.mapCommands) {
+        const CRPCCommand *pcmd = command.second.front();
+        try {
+            pcmd->actor(jreq, unused_result, true);
+        } catch (const std::exception& e) {
+            printf("===\n%s\n", e.what());
+            continue;
+        } catch (const UniValue& e) {
+            printf("===\n%s\n", e.write().c_str());
+            continue;
+        } catch (...) {
+            printf("===\n%s\n", pcmd->name.c_str());
+            fflush(stdout);
+            assert(0);
+        }
+        printf("===\n%s\n", pcmd->name.c_str());
+        fflush(stdout);
+        assert(0);
+    }
+    fflush(stdout);
+    abort();
 #if ENABLE_ZMQ
     RegisterZMQRPCCommands(tableRPC);
 #endif
diff --git a/src/rpc/server.h b/src/rpc/server.h
index 01e855605..130819a56 100644
--- a/src/rpc/server.h
+++ b/src/rpc/server.h
@@ -124,7 +124,7 @@ public:
  */
 class CRPCTable
 {
-private:
+public:
     std::map<std::string, std::vector<const CRPCCommand*>> mapCommands;
 public:
     CRPCTable();

```


</details>

File: src/rpc/util.h [hunk 1]
```diff
@@ -165,7 +165,6 @@ struct RPCArg {
          * When possible, the default value should be specified.
```
[the remark this answers is not in the document]
Comment (fanquake) [reply src/rpc/util.h:162]: Taken your suggestions.

**Comment by kouloumos** [comment]
re-ACK 83f70c8e860556426305b1d70fa0a3276784e360

**Review by aureleoules — APPROVED** [review]
ACK 83f70c8e860556426305b1d70fa0a3276784e360

**Pull request merged**